source/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 add_subdirectory(common)
 add_subdirectory(exe)
 add_subdirectory(server)
+add_subdirectory(zipkin)
```

Comment:
I don't think I would add this as a top level directory. Please move to source/common/tracing/zipkin. There are other things we may eventually want to compile out via various build options that are already in common. We can deal with that at that point.